src/RazorSdk/Targets/Sdk.Razor.CurrentVersion.targets

```diff
@@ -50,6 +50,8 @@ Copyright (c) .NET Foundation. All rights reserved.
         <_TargetingNET50OrLater>true</_TargetingNET50OrLater>
         <_TargetingNET60OrLater>true</_TargetingNET60OrLater>
         <UseRazorSourceGenerator Condition="'$(Language)' == 'C#' AND '$(UseRazorSourceGenerator)' == '' ">true</UseRazorSourceGenerator>
+        <!-- Required to support incremental source generator -->
```

Comment:
@captainsafia is the change correct here?

Reply:
Nope, this can be removed. See https://github.com/dotnet/sdk/pull/20247.